ivy/functional/backends/paddle/layers.py

```diff
@@ -77,18 +77,52 @@ def _pad_before_conv(x, filters, strides, padding, dims, dilations, data_format)
     )
 
 
+@with_unsupported_device_and_dtypes(
+    {"2.5.0 and below": {"cpu": ("float16",)}},
+    backend_version,
+)
 def conv1d(
     x: paddle.Tensor,
     filters: paddle.Tensor,
-    strides: Union[int, Tuple[int]],
-    padding: Union[str, int, Sequence[Tuple[int, int]]],
+    strides: Union[int, Tuple[int]] = 1,
+    padding: Union[str, int, Sequence[Tuple[int, int]]] = 0,
```

Comment:
please don't change anything in the function signature and keep it consistent across the framework

Reply:
Got it 👌